lisa/sut_orchestrator/ready.py

```diff
@@ -68,6 +86,12 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
         # ready platform doesn't support delete environment
         pass
 
+    def delete_environment(self, environment: Environment) -> None:
+        # This is necessary when running multiple test cases that share the same
+        # environment, as it avoids skipping later cases due to cleanup.
+        if not self._ready_runbook.reuse_dirty_env:
```

Comment:
I think we need log a message, so we know an environment is marked as dirty. And the test case may need to be fixed.

Reply:
I don't quite understand the meaning of `the test case may need to be fixed.`, I try to write log lines, please double check, thanks.

Reply:
We should know, when an env is dirty, it mostly like the previous test case damaged the env. We need to check if the test case should be fixed.